frontend/src/containers/RecipePage/Recipe.css

```diff
@@ -0,0 +1,154 @@
+.overall-container {
+  margin: 0px auto;
+  padding: 0px;
+}
+
+.user-edit-button {
+  display: inline-block;
+  float: right;
+  margin: 20px 26px;
+  border: 3px solid #ffa616;
+  border-radius: 13%;
+  color: #ffa616;
+  font-family: 'Nunito', sans-serif;
+  font-size: 14px;
+  height: 4em;
+  padding-left: .5em;
+  padding-right: .8em;
+  line-height: 3em;
+  font-weight: 600;
+  width: 120px;
+  text-align: center;
+  margin-top: 70px;
+}
+
+.recipe-icon {
+  margin-top: 50px;
+  height: 80px;
+  line-height: 80px;
+  text-align: left;
+  font-size: 23.4px;
+  color: white;
+  padding: 0px;
+}
+
+.recipe-photo {
+  width: 90%;
+  height: 500px;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.tags-bar {
+  display: flex;
+  justify-content: center;
+
+}
+
+#recipe-descrip {
+  text-align: center;
+  padding: 1% 3% 3% 3%;
+
+}
+
+#hr {
+  border: 0.5px solid lightgrey;
+  width: 90%;
+  margin: 0 auto;
+}
+
+#ingredient-box {
+  display: flex;
+  justify-content: center;
+  margin-top: 1%;
+  padding: 1% 3% 3% 3%;
+  margin-bottom: -1%;
+}
+
+#ingredient-ul {
+  column-count: 2;
+  column-gap: 50px;
+  padding: 6% 3% 3% 3%;
+  width: 100%;
+
+}
+
+#recipe-title, #direction-title, #ingredient-title {
+  margin-top: 2%;
+  text-align: center;
+  padding: 1% 3% 3% 3%;
+  font-size: 50px;
+  font-weight: bold;
+}
+
+#direction-container {
+  margin-bottom: 20px;
+  text-align: left;
+  padding: 1% 3% 3% 3%;
+}
+
+.single-direction {
+  color: black;
+  text-align: left;
+  margin: 30px 0;
+
+}
+
+.direction-index {
+  display: inline;
+  margin-right: 8px;
+}
+
+.direction-action {
+  display: inline;
+  vertical-align: top;
+  font-size: 25px;
```

Comment:
I think we want the actual directions to the be the same font size as the ingredients and the snippet.

Reply:
changed font size for directions